src/wymeditor/editor/image-handler.js

```diff
@@ -575,6 +579,7 @@ WYMeditor.ImageHandler.prototype._onImgMousedown = function (evt) {
     }
 
     // returning false here prevents drag of image
+    // except for in IE8
```

Comment:
So this doesn't really prevent drag and drop. I didn't try much figuring out a way to prevent drag and drop. I think that the important thing is that it is not suggested in the UI.